cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -30,6 +30,491 @@ Cask Data Application Platform Release Notes
+  its own repository at `github.com/hydrator/kinesis-spark-streaming-source <https://github.com/hydrator/kinesis-spark-streaming-source>`__.
+
+- :cask-issue:`HYDRATOR-552` - Added a plugin for sampling data from a source, available
+  at `github.com/hydrator/sampling-aggregator <https://github.com/hydrator/sampling-aggregator>`__.
+
+- :cask-issue:`HYDRATOR-585` - The HTTP Sink plugin (for posting data from a pipeline to
+  an external endpoint) has been added at `github.com/hydrator/http-sink <https://github.com/hydrator/http-sink>`__.
+
+- :cask-issue:`HYDRATOR-954` - The Kinesis Source plugin now works in realtime pipelines.
+
+- :cask-issue:`HYDRATOR-983` - Added a Feature Generator plugin for a pipeline builder.
+
+- :cask-issue:`HYDRATOR-1049` - Added a DynamoDb Sink as a plugin, available at
+  `github.com/hydrator/dynamodb-sink <https://github.com/hydrator/dynamodb-sink>`__.
+
+- :cask-issue:`HYDRATOR-1050` - Added a DynamoDB Batch Source plugin, available at
+  `github.com/hydrator/dynamodb-source <https://github.com/hydrator/dynamodb-source>`__.
+
+- :cask-issue:`HYDRATOR-1172` - Added a Realtime Stream Source plugin, available at
+  `github.com/hydrator/realtime-stream-source <https://github.com/hydrator/realtime-stream-source>`__.
+
+- :cask-issue:`HYDRATOR-1249` - The Tokenizer plugin is now available in it own repository
+  at `github.com/hydrator/tokenizer-analytics <https://github.com/hydrator/tokenizer-analytics>`__.
+
+- :cask-issue:`HYDRATOR-1250` - The NGramTransform plugin is now available in its own
+  repository at `github.com/hydrator/ngram-analytics <https://github.com/hydrator/ngram-analytics>`__.
+
+- :cask-issue:`HYDRATOR-1251` - The DecisionTree Regression plugins are now available in
+  their own repository at `github.com/hydrator/decision-tree-analytics <https://github.com/hydrator/decision-tree-analytics>`__.
+
+- :cask-issue:`HYDRATOR-1252` - The SKipGram Feature Generator plugin is now available in
+  its own repository at `github.com/hydrator/skipgram-analytics <https://github.com/hydrator/skipgram-analytics>`__.
+
+- :cask-issue:`HYDRATOR-1253` - The Naive Bayes Analytics plugin is now available in its
+  own repository at `github.com/hydrator/naive-bayes-analytics <https://github.com/hydrator/naive-bayes-analytics>`__.
+
+- :cask-issue:`HYDRATOR-1254` - The HashingTF Feature Generator plugin is now available in
+  its own repository at `github.com/hydrator/hashing-tf-feature-generator <https://github.com/hydrator/hashing-tf-feature-generator>`__.
+
+- :cask-issue:`HYDRATOR-1255` - The LogisticRegression plugins are now available in their
+  own repository at `github.com/hydrator/logistic-regression-analytics <https://github.com/hydrator/logistic-regression-analytics>`__.
+
+- :cask-issue:`HYDRATOR-1398` - Support added for Table datasets for lookups in plugins
+  and pipelines.
+
+
+Improvements
+------------
+
+- :cask-issue:`CDAP-3383` - Enabled in CDAP invalid transaction list pruning, a new
```

Comment:
"Enabled in CDAP invalid transaction list pruning, a new
  feature introduced in Apache Tephra. This automates the pruning of the invalid transaction
  list after data for the invalid transaction has been dropped."